datadog-ruby_core_source.gemspec

```diff
@@ -1,13 +1,12 @@
 # encoding: utf-8 -*-
 require 'rubygems' unless defined? Gem
-require File.dirname(__FILE__) + "/lib/debase/ruby_core_source/version"
+require File.dirname(__FILE__) + "/lib/datadog/ruby_core_source/version"
 
 Gem::Specification.new do |s|
-  s.name = "debase-ruby_core_source"
-  s.version = Debase::RubyCoreSource::VERSION
+  s.name = "datadog-ruby_core_source"
+  s.version = Datadog::RubyCoreSource::VERSION
   s.authors = ["Mark Moseley", "Gabriel Horner", "JetBrains RubyMine Team"]
-  s.email = "os97673@gmail.com"
```

Comment:
Do we need a team email address here?

Reply:
https://github.com/DataDog/dd-trace-rb/blob/master/datadog.gemspec uses dev@datadoghq.com so that should be good enough in my opinion if there are no better suggestions.

Reply:
will add dev@datadoghq.com!